eslint/babel-eslint-config-internal/package.json

```
@@ -1,21 +1,15 @@
 {
-  "name": "eslint-config-babel",
-  "version": "9.0.0",
-  "description": "ESLint config for babel",
+  "name": "@babel/eslint-config-internal",
+  "version": "7.7.0",
+  "description": "The Babel Team's ESLint configuration",
```

```suggestion
  "description": "The Babel Team's ESLint configuration. Since it's internal, it might not respect semver.",
```

:stuck_out_tongue: :pray: